netman/adapters/switches/juniper/base.py

```diff
@@ -415,9 +415,7 @@ def edit_interface_spanning_tree(self, interface_id, edge=None):
 
     def set_interface_state(self, interface_id, state):
         update = Update()
-        update.add_interface(interface_main_update(interface_id, [
-            to_ele("<disable />") if state is OFF else to_ele("<enable />")
-        ]))
+        update.add_interface(interface_state_update(interface_id, state))
```

Comment:
If you are setting the state to off then yes, `<disable />` will be in place. If not, then an operation to attempt to remove `<disable />` will be set per: `<disable operation="delete" />`